mantidproject/mantid
# Frequency domain analysis clean up

**Description of work**
This is a clean up of the frequency domain analysis GUI to match the new structure. A few other minor improvements have also been made. 

To have muon analysis 2 as an interface you will need add the following to https://github.com/mantidproject/mantid/blob/master/Framework/Properties/Mantid.properties.template#L23    `Muon/Muon_Analysis_2.py`


**To test:**
Open muon analysis 2 and check it exists. Then you can close it

Open Muon Analysis (v1) and load some data
Close Muon Analysis
Open Frequency Domain analysis and run an FFT 
A workspace will be added to the workspace group, check it has a history
Change the combo box at the top to maxent
Run maxent and check it produces an output

The tables in both maxent and FFT mode should now also be rescalable 



Fixes #22399. 

**Release Notes** 


---
Does not need to be in release notes
#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
